test/routes/paymentCalculation.integration.test.js

```diff
@@ -288,5 +288,65 @@ describe('POST /parcels/{parcelRef/actions/{actionId}/payment-calculation', () =
         )
       })
     })
+
+    describe('with cultivated parcel rule', () => {
+      beforeEach(async () => {
+        jest.mock(
+          '../../data/actions.json',
+          () => [
+            {
+              id: 'SW6',
+              description: 'Winter cover crops',
+              rate: 114,
+              rules: [
+                { id: 8, enabled: true }
+              ]
+            }
+          ]
+        )
+        await setUpServer()
+      })
+
+      test('parcel is eligible if parcel is arable land', async () => {
```

Comment:
This test and all subsequent ones are testing both landCoverClass and area but the names don't reflect this.

Reply:
I've only enabled the cultivated parcel rule, not any area rules, for these tests.

Reply:
I've removed the area calculation from those three tests in favour of passing an arbitrary value. That seems less confusing given that the area rules are disabled for those tests.